distribution/scripts/ballerina/content_based_routing/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ballerina generates this directory during the compilation of a package.
```

Comment:
We can simply want only one gitignore file at root level

Reply:
Updated